debian/rules

```diff
@@ -9,8 +9,6 @@ override_dh_auto_configure:
 override_dh_systemd_start:
 	dh_systemd_start --no-start
 
-# Enable all units here, except swap, since dh-systemd doesn't get the
```

Comment:
now that we don't have swap is it possible to remove the override_dh_systemd_enable section altogether?